extensions/Variables/Variables.php

```diff
@@ -177,7 +177,7 @@ static function pfObj_var( Parser &$parser, $frame, $args) {
 		return $varVal;
     }
 	
-	static function pf_var_final( Parser &$parser, $varName, $defaultVal = '' ) {
+	static function pf_var_final( Parser $parser, $varName, $defaultVal = '' ) {
 		$varStore = self::get( $parser );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$varStore" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)